config/migrations/2024/20241219102800-blankenberge-set-end-date-for.sparql

```diff
@@ -0,0 +1,38 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX dcterms: <http://purl.org/dc/terms/>
+
+DELETE {
+  GRAPH ?g {
+    ?mandataris mandaat:einde ?noEndDate.
+    ?mandataris mandaat:status ?status.
+    ?mandataris dcterms:modified ?modified.
+  }
+}
+INSERT {
+  GRAPH ?g {
+    ?mandataris mandaat:einde """2024-12-15"""^^xsd:date.
+    ?mandataris mandaat:status <http://data.vlaanderen.be/id/concept/MandatarisStatusCode/b8866fa2-d61c-4e3d-afaf-8a29eaaa7fb9>. # Fake mandataris beeindigd status
```

Comment:
this should be the status effectief instead.
Beeindigd is NOT a status, it is only used in the frontend to be able to show something to users when selecting beeindigd. Beeindigd just means the end date is in the past.

Reply:
"Ahzo"

removed setting the status in the query
![image](https://github.com/user-attachments/assets/b23baf65-b537-46af-87ef-cfd8b46749d6)

SOMETHING ELSE: Isn't it strange that this table than shows a different state?
![image](https://github.com/user-attachments/assets/bfc9a60f-6441-4746-a86e-bd756589f979)